src/pages/signup-affiliates/index.tsx

```diff
@@ -1,11 +1,206 @@
 import React from 'react'
+import styled from 'styled-components'
+import ReCAPTCHA from 'react-google-recaptcha'
 import AffiliateSignupLayout, { SignUpWrapper } from './components/_layout'
-import { WithIntl } from 'components/localization'
+import { localize, WithIntl } from 'components/localization'
+import { Input, Button } from 'components/form'
+import { Header, LinkText, LocalizedLinkText } from 'components/elements'
+import device from 'themes/device'
+import Google from 'images/svg/custom/googlenew.svg'
+import Apple from 'images/svg/custom/applenew.svg'
+import Facebook from 'images/svg/custom/facebooknew-blue.svg'
 
-const AffiliateSignup = () => {
+const StyledNote = styled.div`
+    padding: 8px 0;
+    margin: 24px 0 16px;
+    height: 36px;
+    width: 406px;
+    border-radius: 4px;
+    background-color: rgba(248, 248, 249, 1);
```

Comment:
Can we get the hex value of this color? maybe we already have it in our variables.

Reply:
okay